### PR TITLE
transfer sections added

### PR DESCRIPTION
**Pull request by sepehr2github**
- Just all transfer sections added

### CODE REVIEW

File: packages/web-app/src/pages/transfers.tsx [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+        <TransferSectionWrapper title={t('allTransfer.thisWeek')}>
+          <div className="my-2 space-y-2 border-solid">
+            <ColoredDiv />
+            <ColoredDiv />
+            <ColoredDiv />
+          </div>
+        </TransferSectionWrapper>
+      </SectionContainer>
+      <SectionContainer>
+        <TransferSectionWrapper title={'November'}>
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/transfers.tsx:22]: I think this should be computed dynamically.

Also, this must be translatable to different language.

File: packages/web-app/src/pages/transfers.tsx [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+          <div className="my-2 space-y-2 border-solid">
+            <ColoredDiv />
+            <ColoredDiv />
+            <ColoredDiv />
+          </div>
+        </TransferSectionWrapper>
+      </SectionContainer>
+      <SectionContainer>
+        <TransferSectionWrapper title={'2021'}>
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/transfers.tsx:31]: Same here

File: packages/web-app/src/pages/transfers.tsx (hunk 1 above)
Comment (WissenIstNacht) [inline packages/web-app/src/pages/transfers.tsx:22]: I think this should be computed dynamically.

Also, this must be translatable to different language.

File: packages/web-app/src/pages/transfers.tsx (hunk 2 above)
Comment (WissenIstNacht) [inline packages/web-app/src/pages/transfers.tsx:31]: Same here

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Thanks mate. 🙂 

I think it should already be possible to set the sections titles dynamically (even if the data is not present. Maybe a hook that, given a series of dates, divides them into last week, last month, last year, older?

File: packages/web-app/src/pages/transfers.tsx [hunk 3]
```diff
@@ -1,7 +1,50 @@
 import React from 'react';
+import styled from 'styled-components';
+import {useTranslation} from 'react-i18next';
+
+import {TransferSectionWrapper} from 'components/sectionWrapper';
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/transfers.tsx:5]: If we are using these guys outside of the finance overview page, we could combine them into one SectionWrapper or ListGroup component that shows IconButton and SeeAllButton conditionally. That said, this probably shouldn't be done right now.

File: packages/web-app/src/pages/transfers.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/pages/transfers.tsx:22]: Agreed. There was a thread about a formatting and date library. Lets see if we can agree on one quickly so we can use it there.

File: packages/web-app/src/pages/transfers.tsx [hunk 4]
```diff
@@ -1,7 +1,50 @@
...
+          </div>
+        </TransferSectionWrapper>
+      </SectionContainer>
+    </div>
+  );
 };
 
 export default Transfers;
+
+const ColoredDiv = styled.div.attrs({className: 'h-6 w-full bg-blue-100'})``;
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/transfers.tsx:48]: `blue-100` isn't part of the design system, is it?

**Review by Fabricevladimir — COMMENTED** [review]
Looks good. We'll be able to flesh out the page easily now.

File: packages/web-app/src/pages/transfers.tsx (hunk 4 above)
Comment (WissenIstNacht) [reply packages/web-app/src/pages/transfers.tsx:48]: I think he copy pasted from my PR a while back, sorry for that. 😅

That being, said their just placeholders 👍🏼

Comment (sepehr2github) [reply packages/web-app/src/pages/transfers.tsx:48]: @Fabricevladimir, I copied from @WissenIstNacht as a placeholder. All the ColoredDiv will remove soon

Comment (Fabricevladimir) [reply packages/web-app/src/pages/transfers.tsx:48]: Pffft! I didn't even realize that it was for the placeholder. This one is on me

File: packages/web-app/src/pages/transfers.tsx (hunk 1 above)
Comment (sepehr2github) [reply packages/web-app/src/pages/transfers.tsx:22]: I had the same idea but due to lack of transfers data. It's not clear to me what do you expect?
@Fabricevladimir 
@WissenIstNacht

File: packages/web-app/src/pages/transfers.tsx (hunk 3 above)
Comment (sepehr2github) [reply packages/web-app/src/pages/transfers.tsx:5]: Good idea. We will definitely do that

File: packages/web-app/src/pages/transfers.tsx (hunk 1 above)
Comment (WissenIstNacht) [reply packages/web-app/src/pages/transfers.tsx:22]: What would you do if you had data? :)

Essentially, there's two things that need to be done here: 

1. Determine what is the last week, month, year (and/or maybe older). Note, this has nothing to do with transfer data being available.
2. Categorize data according to the answers above. Note, this does, but you should be able to come up with some data that allows you to at least test this.

The first step could be done with a simple `utils` type of function that returns the date that delimit the categories (`getDateCategories: () => dates[]`). The second could be done with a hook that takes transfer data and splits them up into these categories (`useCategorizedTransfers: (TransferData[]) => TransferData[][]`). For this step, I think we'll have to assume that there will be some type like `TransderData` that will contains some form of date.

**Comment by WissenIstNacht** [comment]
Hey @sepehr2github why did you close this PR? 😅

**Comment by sepehr2github** [comment]
> Hey @sepehr2github why did you close this PR? 😅

Because I want to open another PR. :)))

**Comment by WissenIstNacht** [comment]
But you can have several open PRs ...? :)

Are you going to continue working on this later or your not going to be working on this anymore?

**Pull request closed without merging**